tests/test_restore.py

```diff
@@ -21,15 +26,80 @@ def test_positive_restore_online_backup(ansible_module):
 
     :CaseImportance: Critical
     """
+    # preparing a target dir
+    backup_dir = 'online_backup_restore'
+    stp = ansible_module.command("rm -rf /tmp/{}".format(backup_dir))
+    print(stp.values())
+    assert stp.values()[0]["rc"] == 0
+    stp = ansible_module.command("mkdir /tmp/{}".format(backup_dir))
+    assert stp.values()[0]["rc"] == 0
+    stp = ansible_module.command(
+        "chown -R postgres /tmp/{}".format(backup_dir))
```

Comment:
I had to do these 3 ansible command runs, because file module seems not supported by pytest-ansible, when trying ti use it I got:

```
E               ansible.errors.AnsibleParserError: this task 'file' has extra params, which is only allowed in the following modules: set_fact, include_tasks, include, script, import_role, group_by, import_tasks, command, win_command, add_host, raw, meta, include_vars, win_shell, shell, include_role

../../Envs/testfm-py3-master/lib/python3.6/site-packages/ansible/parsing/mod_args.py:316: AnsibleParserError

```
So maybe we are bit too far on the bleeding edge here, alternative would be to pull in paramiko, but I wanted to use what we already have imported, at least for now

Reply:
It should work..else we can file issue at https://github.com/ansible/pytest-ansible/issues